src/main/java/org/jitsi/videobridge/xmpp/XmppCommon.java

```diff
@@ -214,6 +239,16 @@ else if (request instanceof ShutdownIQ)
         {
             response = null;
         }
+
+        if (delayStats != null)
+        {
+            long delay = System.currentTimeMillis() - start;
+            if (delay > 100)
+            {
+                logger.warn("Took " + delay + "ms to handle IQ: " + request.toXML());
+            }
```

Comment:
Missing space before `"ms`?

Reply:
No, that was intentional. I've *always* formatted it this way! Looks like I've always been wrong :)